### PR TITLE
[4.0] remove slash

### PR DESCRIPTION
**Pull request by brianteeman**
continue the removal of closing slashes for tags that do not require this in html5

@dgt41 I spotted several changes that would need to be made in the JS too but I have not touched them

admin-article-pagebreak.js
com_finder/js etc

### CODE REVIEW

**Comment by dgt41** [comment]
> `admin-article-pagebreak.js`


This is a convention so:
- the first part specifies if the script is used in backend or front end
- the second part specifies the view
- the third part specifies the layout

By the way I didn't came up with this, but it really makes sense...

**Comment by brianteeman** [comment]
i am not set up for running grunt etc right now. will take me a few days till i have the time to do that

**Comment by brianteeman** [comment]
@dgt41 hopefully I did the JS correctly

File: media/com_content/js/admin-article-readmore.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ window.insertReadmore = function(editor) {
 		content = (new Function('return ' + options.editor))();
 	}
 
-	if (content.match(/<hr\s+id=("|')system-readmore("|')\s*\/*>/i)) {
+	if (content.match(/<hr\s+id=("|')system-readmore("|')\s*>/i)) {
```
Comment (dgt41) [inline media/com_content/js/admin-article-readmore.js]: Do you want to remove this part `\/*`?
I think it covers cases like `<hr id="system-readmore" style="color:black">`

**Comment by dgt41** [comment]
looks fine, you have one conflict tho

File: media/com_content/js/admin-article-readmore.js (hunk 1 above)
Comment (brianteeman) [inline media/com_content/js/admin-article-readmore.js]: i shall put it back then - i am not great on regex

Comment (brianteeman) [inline media/com_content/js/admin-article-readmore.js]: reverted that change

**Comment by brianteeman** [comment]
Conflicts resolved

File: media/com_content/js/admin-article-readmore.js (hunk 1 above)
Comment (dgt41) [inline media/com_content/js/admin-article-readmore.js]: I'm not a regex expert either, so I run this and the `/*` means that the tags can have more than one space before the closing tag. 
screen shot 2017-05-08 at 13 53 31

Core will be ok with your code, I'm not sure tho if any 3rd PD editor is doing something stupid (like 2 spaces).

**Comment by C-Lodder** [comment]
@wilsonge - good to merge on code review.

File: plugins/content/pagebreak/pagebreak.php [hunk 2]
```diff
@@ -16,15 +16,15 @@
 /**
  * Page break plugin
  *
- * <strong>Usage:</strong>
- * <code><hr class="system-pagebreak" /></code>
- * <code><hr class="system-pagebreak" title="The page title" /></code>
+ * <strong>Usage:</srong>
```
Comment (wilsonge) [inline plugins/content/pagebreak/pagebreak.php]: This looks like a incorrect change

Comment (brianteeman) [inline plugins/content/pagebreak/pagebreak.php]: thats what happens when I try to work on merge conflicts :( 

Fixed now

**Comment by brianteeman** [comment]
grrh merge conflicts

**Comment by tonypartridge** [comment]
Showing we have conflicts here..

**Comment by brianteeman** [comment]
@wilsonge i can redo all of this bu it will need you to merge it reasonably quickly or it will go out of sync again due to the js changes

**Comment by wilsonge** [comment]
That's fine. Sorry you didn't comment after you fixed the typo so I didn't get a notification

**Comment by brianteeman** [comment]
Closed. I really cant be bothered to have anything to do with js in j4

**Comment by C-Lodder** [comment]
Brian, keep it open and I'll push a PR to your repo after lunch with the fixes

**Comment by brianteeman** [comment]
no its deleted now

**Pull request closed without merging**